docs/components/beacon/index.html

```diff
@@ -0,0 +1,77 @@
+---
+title: Beacon
+minver: 0.22.0
+also:
+  elements/hx-beacon: <hx-beacon>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    A beacon is an animation that attracts a user's attention
+    to a new feature or advanced functionality in a control panel.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="single-beacon">Beacon</h2>
+      <p>
+        The custom <code>{{page.title}}</code> element provides an animation
+        that attracts a user's attention to a new feature
+        or advanced functionality in a control panel. A beacon disappears after a user clicks it.
+      </p>
+    </header>
+
+    <div class="example" id="vue-beaconDemo" v-cloak>
+      <div>
+        <hx-beacon
+          @dismiss="onEvent"
+        >
+        </hx-beacon>
+      </div>
+
+      <footer>
+        <pre><code v-text="snippet"></code></pre>
+     </footer>
+    </div>
+  </section>
+
+
+  <section>
+    <header>
+      <h2 id="basic-menu">Working Example</h2>
+      {# TODO: add section description #}
+    </header>
+
```

Comment:
Please add a descriptive paragraph after the `<header>` block:

```html
<p>
  This is an example implementation of `<hx-beacon>`.   We leave it to HelixUI implementors
  to decide what works best for their applications.
</p>
```

Reply:
Changes have been done, please review